docs/en/sql-reference/functions/string-search-functions.md

```diff
@@ -378,6 +378,57 @@ For other regular expressions, the code is the same as for the ‘match’ funct
 
 The same thing as ‘like’, but negative.
 
+## iLike {#ilike}
+
+Case insensitive variant of function `like`. It should respect UTF-8 and work similar to `positionCaseInsensitiveUTF8`.
```

Comment:
If we refer to something, for example functions, we need to provide links. 
I have understood nothing about UTF-8 reference.